2018-12-31T12:58:12.410+0800 I CONTROL  [main] Automatically disabling TLS 1.0, to force-enable TLS 1.0 specify --sslDisabledProtocols 'none'
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] MongoDB starting : pid=74118 port=27017 dbpath=/Users/yomi/NodeJS/Simple-SocketIO-Server/db_mongodb 64-bit host=yomideMacBook-Pro.local
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] db version v4.0.5
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] git version: 3739429dd92b92d1b0ab120911a23d50bf03c412
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] allocator: system
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] modules: none
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] build environment:
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten]     distarch: x86_64
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten]     target_arch: x86_64
2018-12-31T12:58:12.422+0800 I CONTROL  [initandlisten] options: { storage: { dbPath: "/Users/yomi/NodeJS/Simple-SocketIO-Server/db_mongodb" }, systemLog: { destination: "file", path: "/Users/yomi/NodeJS/Simple-SocketIO-Server/db_mongodb/mongod.log" } }
2018-12-31T12:58:12.422+0800 I STORAGE  [initandlisten] Detected data files in /Users/yomi/NodeJS/Simple-SocketIO-Server/db_mongodb created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-12-31T12:58:12.423+0800 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-12-31T12:58:13.080+0800 I STORAGE  [initandlisten] WiredTiger message [1546232293:80114][74118:0x1181755c0], txn-recover: Main recovery loop: starting at 5/6400 to 6/256
2018-12-31T12:58:13.159+0800 I STORAGE  [initandlisten] WiredTiger message [1546232293:159197][74118:0x1181755c0], txn-recover: Recovering log 5 through 6
2018-12-31T12:58:13.211+0800 I STORAGE  [initandlisten] WiredTiger message [1546232293:211669][74118:0x1181755c0], txn-recover: Recovering log 6 through 6
2018-12-31T12:58:13.252+0800 I STORAGE  [initandlisten] WiredTiger message [1546232293:252559][74118:0x1181755c0], txn-recover: Set global recovery timestamp: 0
2018-12-31T12:58:13.303+0800 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] 
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] 
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-12-31T12:58:13.333+0800 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-12-31T12:58:13.334+0800 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-12-31T12:58:13.334+0800 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-12-31T12:58:13.334+0800 I CONTROL  [initandlisten] 
2018-12-31T12:58:13.334+0800 I CONTROL  [initandlisten] 
2018-12-31T12:58:13.334+0800 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-12-31T12:58:13.371+0800 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/Users/yomi/NodeJS/Simple-SocketIO-Server/db_mongodb/diagnostic.data'
2018-12-31T12:58:13.374+0800 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-12-31T12:58:39.135+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51587 #1 (1 connection now open)
2018-12-31T12:58:39.136+0800 I NETWORK  [conn1] received client metadata from 127.0.0.1:51587 conn1: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "4.0.5" }, os: { type: "Darwin", name: "Mac OS X", architecture: "x86_64", version: "18.2.0" } }
2018-12-31T12:59:00.130+0800 I ACCESS   [conn1] Supported SASL mechanisms requested for unknown user 'dba@admin'
2018-12-31T12:59:00.135+0800 I ACCESS   [conn1] SASL SCRAM-SHA-1 authentication failed for dba on admin from client 127.0.0.1:51587 ; UserNotFound: Could not find user dba@admin
2018-12-31T12:59:28.803+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51605 #2 (2 connections now open)
2018-12-31T12:59:28.808+0800 I NETWORK  [conn2] received client metadata from 127.0.0.1:51605 conn2: { driver: { name: "nodejs", version: "3.1.9" }, os: { type: "Darwin", name: "darwin", architecture: "x64", version: "18.2.0" }, platform: "Node.js v8.9.3, LE, mongodb-core: 3.1.8" }
2018-12-31T12:59:28.899+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51607 #3 (3 connections now open)
2018-12-31T12:59:28.982+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51609 #4 (4 connections now open)
2018-12-31T12:59:28.989+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51611 #5 (5 connections now open)
2018-12-31T12:59:38.212+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51613 #6 (6 connections now open)
2018-12-31T13:01:15.275+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51641 #7 (7 connections now open)
2018-12-31T13:01:15.280+0800 I NETWORK  [conn7] received client metadata from 127.0.0.1:51641 conn7: { driver: { name: "nodejs", version: "3.1.10" }, os: { type: "Darwin", name: "darwin", architecture: "x64", version: "18.2.0" }, platform: "Node.js v8.11.3, LE, mongodb-core: 3.1.9" }
2018-12-31T13:01:15.280+0800 I ACCESS   [conn7] Supported SASL mechanisms requested for unknown user 'user1@admin'
2018-12-31T13:01:15.288+0800 I ACCESS   [conn7] SASL SCRAM-SHA-1 authentication failed for user1 on admin from client 127.0.0.1:51641 ; UserNotFound: Could not find user user1@admin
2018-12-31T13:01:15.290+0800 I NETWORK  [conn7] end connection 127.0.0.1:51641 (6 connections now open)
2018-12-31T13:05:51.331+0800 I STORAGE  [conn1] createCollection: admin.system.users with generated UUID: 852125c3-dda3-4b03-9468-6ed4ef6f03a0
2018-12-31T13:05:51.393+0800 I COMMAND  [conn1] command admin.$cmd appName: "MongoDB Shell" command: createUser { createUser: "user1", pwd: "xxx", roles: [ { role: "readWrite", db: "Test" } ], digestPassword: true, writeConcern: { w: "majority", wtimeout: 600000.0 }, lsid: { id: UUID("bd4bf4c3-2727-4be6-b2b9-43f1249dbf92") }, $db: "admin" } numYields:0 reslen:38 locks:{ Global: { acquireCount: { r: 4, w: 4 } }, Database: { acquireCount: { W: 4 } }, Collection: { acquireCount: { w: 3 } } } protocol:op_msg 103ms
2018-12-31T13:06:05.307+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51736 #8 (7 connections now open)
2018-12-31T13:06:05.312+0800 I NETWORK  [conn8] received client metadata from 127.0.0.1:51736 conn8: { driver: { name: "nodejs", version: "3.1.10" }, os: { type: "Darwin", name: "darwin", architecture: "x64", version: "18.2.0" }, platform: "Node.js v8.11.3, LE, mongodb-core: 3.1.9" }
2018-12-31T13:06:05.342+0800 I ACCESS   [conn8] Successfully authenticated as principal user1 on admin
2018-12-31T13:06:05.358+0800 I STORAGE  [conn8] createCollection: TestDB.TestTable2 with generated UUID: 4a3b6388-13fe-40c3-a1eb-b64a0b7e1bfc
2018-12-31T13:06:05.368+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51738 #9 (8 connections now open)
2018-12-31T13:06:05.372+0800 I ACCESS   [conn9] Successfully authenticated as principal user1 on admin
2018-12-31T13:06:05.375+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51740 #10 (9 connections now open)
2018-12-31T13:06:05.382+0800 I ACCESS   [conn10] Successfully authenticated as principal user1 on admin
2018-12-31T13:06:05.384+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51742 #11 (10 connections now open)
2018-12-31T13:06:05.390+0800 I ACCESS   [conn11] Successfully authenticated as principal user1 on admin
2018-12-31T13:06:05.393+0800 I NETWORK  [listener] connection accepted from 127.0.0.1:51744 #12 (11 connections now open)
2018-12-31T13:06:05.397+0800 I ACCESS   [conn12] Successfully authenticated as principal user1 on admin
2018-12-31T13:06:05.407+0800 I INDEX    [conn8] build index on: TestDB.TestTable2 properties: { v: 2, key: { column6: 1 }, name: "column6_1", ns: "TestDB.TestTable2" }
2018-12-31T13:06:05.407+0800 I INDEX    [conn8] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2018-12-31T13:06:05.413+0800 I INDEX    [conn8] build index done.  scanned 0 total records. 0 secs
2018-12-31T13:06:32.683+0800 I NETWORK  [conn1] end connection 127.0.0.1:51587 (10 connections now open)
2018-12-31T13:08:05.463+0800 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-12-31T13:08:05.463+0800 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-12-31T13:08:05.463+0800 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-12-31T13:08:05.464+0800 I CONTROL  [signalProcessingThread] Shutting down free monitoring
2018-12-31T13:08:05.464+0800 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-12-31T13:08:05.466+0800 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-12-31T13:08:05.604+0800 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-12-31T13:08:05.605+0800 I CONTROL  [signalProcessingThread] now exiting
2018-12-31T13:08:05.605+0800 I CONTROL  [signalProcessingThread] shutting down with code:0
